.babelrc

```diff
@@ -0,0 +1,54 @@
+{ 
+  "env": {
+    "commonjs": {
+      "presets": ["es2015"]
```

Comment:
Shared between all, probably fine to put it at the top level? Or does it have to go along with the `plugins` config?